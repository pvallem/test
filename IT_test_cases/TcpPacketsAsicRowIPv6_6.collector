bidir_flow_info {
  key {
    src_address: "\001\002\003\001\000\000\000\000\000\000\000\000\000\000\000\000"
    dst_address: "\t@\000\000\000\000\000\000\000\000\000\000\000\000\000\000"
    src_port: 3657
    dst_port: 222
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV6
  }
  flow_info {
    start_time: 1388563200000010
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 2
    }
    flow_features {
      packet_count: 2
    }
    end_time: 1388563200150010
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\001\002\003\000\000\000\000\000\000\000\000\000\000\000\000\000"
    dst_address: "\025\337\000\000\000\000\000\000\000\000\000\000\000\000\000\000"
    src_port: 3657
    dst_port: 222
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: IPV6
  }
  flow_info {
    start_time: 1388563200000020
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 2
    }
    flow_features {
      packet_count: 2
    }
    end_time: 1388563200150020
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\001\002\003\000\000\000\000\000\000\000\000\000\000\000\000\000"
    dst_address: "\003\\\000\000\000\000\000\000\000\000\000\000\000\000\000\000"
    src_port: 3657
    dst_port: 222
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV6
  }
  flow_info {
    start_time: 1388563200000000
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 2
    }
    flow_features {
      packet_count: 2
    }
    end_time: 1388563200150000
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\001\002\003\001\000\000\000\000\000\000\000\000\000\000\000\000"
    dst_address: "\037\303\000\000\000\000\000\000\000\000\000\000\000\000\000\000"
    src_port: 3657
    dst_port: 222
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000030
    key_type: IPV6
  }
  flow_info {
    start_time: 1388563200000030
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 2
    }
    flow_features {
      packet_count: 2
    }
    end_time: 1388563200150030
    sensor_id: "127.0.0.1:5675"
  }
}
