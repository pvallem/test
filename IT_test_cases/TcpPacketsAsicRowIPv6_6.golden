flow_info {
  key {
    src_address: "\001\002\003\000\000\000\000\000\000\000\000\000\000\000\000\000"
    dst_address: "\003\\\000\000\000\000\000\000\000\000\000\000\000\000\000\000"
    src_port: 3657
    dst_port: 222
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV6
  }
  start_time: 1388563200000000
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 2
  }
  end_time: 1388563200050000
}
flow_info {
  key {
    src_address: "\001\002\003\001\000\000\000\000\000\000\000\000\000\000\000\000"
    dst_address: "\t@\000\000\000\000\000\000\000\000\000\000\000\000\000\000"
    src_port: 3657
    dst_port: 222
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV6
  }
  start_time: 1388563200000010
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 2
  }
  end_time: 1388563200050010
}
flow_info {
  key {
    src_address: "\001\002\003\000\000\000\000\000\000\000\000\000\000\000\000\000"
    dst_address: "\025\337\000\000\000\000\000\000\000\000\000\000\000\000\000\000"
    src_port: 3657
    dst_port: 222
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: IPV6
  }
  start_time: 1388563200000020
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 2
  }
  end_time: 1388563200050020
}
flow_info {
  key {
    src_address: "\001\002\003\001\000\000\000\000\000\000\000\000\000\000\000\000"
    dst_address: "\037\303\000\000\000\000\000\000\000\000\000\000\000\000\000\000"
    src_port: 3657
    dst_port: 222
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000030
    key_type: IPV6
  }
  start_time: 1388563200000030
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 2
  }
  end_time: 1388563200050030
}
flow_info {
  key {
    src_address: "\001\002\003\000\000\000\000\000\000\000\000\000\000\000\000\000"
    dst_address: "\003\\\000\000\000\000\000\000\000\000\000\000\000\000\000\000"
    src_port: 3657
    dst_port: 222
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV6
  }
  start_time: 1388563200100000
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 2
  }
  end_time: 1388563200150000
}
flow_info {
  key {
    src_address: "\001\002\003\001\000\000\000\000\000\000\000\000\000\000\000\000"
    dst_address: "\t@\000\000\000\000\000\000\000\000\000\000\000\000\000\000"
    src_port: 3657
    dst_port: 222
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV6
  }
  start_time: 1388563200100010
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 2
  }
  end_time: 1388563200150010
}
flow_info {
  key {
    src_address: "\001\002\003\000\000\000\000\000\000\000\000\000\000\000\000\000"
    dst_address: "\025\337\000\000\000\000\000\000\000\000\000\000\000\000\000\000"
    src_port: 3657
    dst_port: 222
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: IPV6
  }
  start_time: 1388563200100020
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 2
  }
  end_time: 1388563200150020
}
flow_info {
  key {
    src_address: "\001\002\003\001\000\000\000\000\000\000\000\000\000\000\000\000"
    dst_address: "\037\303\000\000\000\000\000\000\000\000\000\000\000\000\000\000"
    src_port: 3657
    dst_port: 222
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000030
    key_type: IPV6
  }
  start_time: 1388563200100030
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 2
  }
  end_time: 1388563200150030
}
